tests/functional_tests/iterator_tests/test_builtins.py

```diff
@@ -108,6 +110,168 @@ def test_arithmetic_and_logical_builtins(backend, builtin, inputs, expected):
         assert np.allclose(np.asarray(out), expected)
 
 
+@pytest.mark.parametrize(
+    "math_fun, ref_impl, inputs",
+    [
+        # FIXME(ben): what about pow?
+        # FIXME(ben): dataset is missing invalid ranges (mostly nan outputs)
+        # FIXME(ben): we're not properly testing different datatypes
+        (
+            abs,
+            np.abs,
+            ([-1, 1, -1.0, 1.0, 0, -0, 0.0, -0.0],),
+        ),
+        (
+            min,
+            # FIXME(ben): what's the signature & semantics of `min` & `max`?
+            # `np.min` and python's built-in `min` have very different signatures...
```

Comment:
Not sure about this one. Python has `min(<iterable>)` and numpy: `np.min(<array>, axis=...)`. So in numpy, you can't do `np.min(a, b)`.

Reply:
In addition to `np.min` there is `np.minimum(a, b)` (https://numpy.org/doc/stable/reference/generated/numpy.minimum.html), which is what we want. So to stay consistent with numpy I would rename to the latter.